BookingsApi/BookingsApi.DAL/Exceptions/JudiciaryPersonNotFoundException.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+#pragma warning disable S3925 // "ISerializable" should be implemented correctly
```

Comment:
we can fix this? :)
see example from notificationapi here
https://github.com/hmcts/vh-notification-api/blob/04b55c73ece01421bcfa14f3243beab69e297507/NotificationApi/NotificationApi.DAL/Exceptions/NotificationNotFoundException.cs

Reply:
Nice one... DONE!